lib/plugins/aws/package/compile/events/stream/index.test.js

```diff
@@ -163,7 +164,11 @@ describe('AwsCompileStreamEvents', () => {
       expect(
         awsCompileStreamEvents.serverless.service.provider.compiledCloudFormationTemplate.Resources
           .FirstEventSourceMappingDynamodbFoo.DependsOn
-      ).to.equal(roleLogicalId);
+      ).to.include(roleLogicalId);
+      expect(
+        awsCompileStreamEvents.serverless.service.provider.compiledCloudFormationTemplate.Resources
+          .FirstEventSourceMappingDynamodbFoo.DependsOn
+      ).to.have.lengthOf(1);
```

Comment:
I wouldn't touch those tests (Ideally all such tests should be refactored to rely on `runServelress`, but ofc that's not in a scope of this PR).

They're pain to work with on any refactors, so adding to them increases maintenance burden

Reply:
It's a slight adjustment to them as to make the things a bit easier I defaulted to have `dependsOn` as an array instead of a string if it's depending on just one resource. What would you suggest here then?

Reply:
> It's a slight adjustment to them as to make the things a bit easier I defaulted to have dependsOn as an array instead of a string if it's depending on just one resource. What would you suggest here then?

Update existing tests, that started to fail and need to be updated cause of that change, but do not introduce a new one

>  I felt like refactoring these tests is not worth it at the moment

Yes, it could be time taking. At some point it'll be good if we do it though, as it'll allow to do greater internal rerfactors without a need of doing full test rewrite.

Reply:
Adjusted :100: 

> Yes, it could be time taking. At some point it'll be good if we do it though, as it'll allow to > do greater internal rerfactors without a need of doing full test rewrite.

I totally agree with that approach, I just had in mind this specific test suite, as if #8137 will be implemented, `stream` will be eventually removed